SPECS-EXTENDED/gpsbabel/gpsbabel.spec

```
@@ -0,0 +1,345 @@
+%global gittag 1_8_0
+
+Summary:        A tool to convert between various formats used by GPS devices
+Name:           gpsbabel
+Version:        1.8.0
+Release:        4%{?dist}
+License:        GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.gpsbabel.org
+# Upstream's website hides tarball behind some ugly php script
+# Original repo is at https://github.com/gpsbabel/gpsbabel
+Source0:        https://github.com/GPSBabel/gpsbabel/archive/refs/tags/%{name}_1_8_0.tar.gz#/%{name}-%{version}.tar.gz
+Source2:        %{name}.png
+# No automatic phone home by default (RHBZ 668865)
+Patch2:         0002-No-solicitation.patch
+
+%ifarch %{qt5_qtwebengine_arches}
+# HACK: Don't build GUI on archs not supported by qtwebengine
+%global build_gui 1
+BuildRequires:  qt5-qtwebchannel-devel
+BuildRequires:  qt5-qtwebengine-devel
+%endif
+BuildRequires:  cmake
+BuildRequires:  desktop-file-utils
+BuildRequires:  libusb1-devel
+BuildRequires:  qt5-qtbase-devel
+BuildRequires:  qt5-qtserialport-devel
+BuildRequires:  shapelib-devel
+BuildRequires:  zlib-devel
+
+%description
+Converts GPS waypoint, route, and track data from one format type
+to another.
+
+%if 0%{?build_gui}
+%package gui
+Summary:        Qt GUI interface for GPSBabel
+Requires:       %{name} = %{version}-%{release}
+
+%description gui
+Qt GUI interface for GPSBabel
+%endif
+
+%prep
+%setup -q -n %{name}-%{name}_%{gittag}
+
+%patch2 -p1
```

Consider `%autosetup`:

```suggestion
%autosetup -n %{name}-%{name}_%{gittag} -p1
```